src/main/java/org/fundacionjala/coding/jimmy/Kata2.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.coding.jimmy;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+/**
+ * This kata was performed by Jimmy Romero.
+ */
+public final class Kata2 {
+
+    private static final String BLANK_SPACE = " ";
+    private static final int MINIMUM_LENGTH_OF_WORD = 2;
+
+    /**
+     * @param stringWords is the complete string which is going to change its inner.
+     * @return the inner of the word sorted.
+     */
+    public String sortInnerContent(String stringWords) {
+        return Arrays.stream(stringWords.split(BLANK_SPACE)).map(w -> sortWord(w)).collect(Collectors.joining(" "));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AIndentationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)

Reply:
that means 
```java
        return Arrays.stream(stringWords.split(BLANK_SPACE))
                .map(this::sortWord)
                .collect(Collectors.joining(" "));

```